traffic_ops/app/lib/UI/DeliveryService.pm

```diff
@@ -587,6 +589,21 @@ sub header_rewrite {
 			$insert->insert();
 			$param_id = $insert->id;
 		}
+
```

Comment:
all of this extra code in this file just to create a new param and link it to all the relevant profiles which, not to mention, creates a lot more queries to the database....imo it's worth exploring the idea of creating a configfiles table instead of using profiles/parameters for location and scope...just my opinion